pkg/service/cluster.go

```diff
@@ -625,6 +629,23 @@ func (s *clusterService) Delete(ctx context.Context, cluster *infrav3.Cluster) e
 
 	_log.Infow("deleting cluster", "name", cluster.Metadata.Name)
 
+	if err != nil {
+		return err
+	}
+
+	//TODO
+	in := &sentryrpc.GetForClusterRequest{
+		Namespace:  "paralus-system",
+		SystemUser: false,
+	}
+	kss := NewKubeconfigSettingService(s.db)
+	var pf cryptoutil.PasswordFunc
+	config, err := kubeconfig.GetConfigForCluster(ctx, s.bs, in, pf, kss, kubeconfig.ParalusSystem)
```

Comment:
sharing as reference of how this rpc is invoked from prompt

https://github.com/paralus/prompt/blob/main/debug/handler.go#L84